tt_eager/tt_dnn/op_library/run_operation.hpp

```diff
@@ -348,6 +348,13 @@ inline std::vector<Tensor> run_with_autoformat(
     return run_with_autoformat(operation, input_tensors, input_formatting, output_layouts, optional_input_tensors, optional_input_formatting);
 }
 
+void launch_op(
+    std::function<Tensor(const std::vector<Tensor>&, const std::vector<std::optional<const Tensor>>&)> op_func,
+    const std::vector<Tensor> input_tensors,
+    Tensor output_tensor,
```

Comment:
Can you pass in a vector `output_tensors`?

Reply:
Changes applied to latest commit. We now have a vector of tensors being passed between the op runner and launch_op API. I think pretty much every op today returns a single tensor, so we just do something like: `return output_tensors.at(0)`in the op runner itself.